index.html

```diff
@@ -297,5 +297,15 @@
   <!-- Game Tiles  -->
   <div id="tiles"></div>
 
+
+  <script>
+    // Purpose: For Preventing overlapping of Menubar on the top-left 
+    // and the logo. 
+      $(document).ready(function(){
+        if($(window).width()<993){
+          $(".brand-logo").removeClass("left");
```

Comment:
Putting style into JavaScript is nothing I approve.
https://github.com/fossasia/labyrinth/pull/695#issuecomment-360567855
If you find another solution, I am happy to merge.
Now, I will need 1-2 reviews of other contributors who approve of this.

Would it be to always center this? 
at width 1012, the logo is not overlapping the menu
at width 960, the logo overlaps as you show.
So: I conclude there are specified behavior for this.
To add more code to remove what is wrong will leave us with an undetected broken code and two peaces of code which could be removed and are not understood as belonging together.

What do you think?

Reply:
1. Yeah, its kind of a skewed version + I like working with Js.
2. I totally agree.
I could maybe make it work with css but I would have to override some defaults of materialise css by using !important. How would that be for a solution.

Reply:
Modify materialize.css. Its license allows us to do that.

I opposed putting style into JavaScript back in #695 as well. I really don't see it as a good practice.

I'll work on fixing all that as soon as I get time. Please don't elevate this problem further.

Read this : https://ilikekillnerds.com/2015/10/css-in-js-its-a-trap/

Any developer community would say this is a bad thing.

Reply:
@himanshuc3 Keeping it in CSS is fine with me.
Then, maybe you can modify the HTML DOM, too - remove and add classes as it fits your need.
@duttaditya18 Thanks for the link.